libraries/AP_HAL_ChibiOS/hwdef/CubeOrange-Octano/defaults.parm

```diff
@@ -0,0 +1,246 @@
+INS_LOG_BAT_MASK,5
+INS_LOG_BAT_OPT,4
+INS_POS1_X,0.82
+INS_POS2_X,0.82
+INS_POS3_X,0.82
+KFF_RDDRMIX,1
+NAVL1_LIM_BANK,35 # Set to match ROLL_LIMIT_DEG
+NAVL1_PERIOD,27
+PTCH_RATE_D,0.0065
+PTCH_RATE_FF,0.7402687
+PTCH_RATE_FLTD,11
+PTCH_RATE_FLTT,3.183099
+PTCH_RATE_I,0.7402687 # DISABLE_CHECKS: it is recommended for RATE_I to match RATE_FF on planes, but the metadata gives a limit of 0.6 for this and 3.0 for FF
+PTCH_RATE_IMAX,0.4
+PTCH_RATE_P,0.2194594
+PTCH_RATE_SMAX,110
+PTCH_TRIM_DEG,3
+PTCH2SRV_RLL,1.05
+PTCH2SRV_RMAX_DN,45
+PTCH2SRV_RMAX_UP,75
+Q_A_ACCEL_P_MAX,20000
+Q_A_ACCEL_R_MAX,20000
+Q_A_ACCEL_Y_MAX,2250
+Q_A_ANG_YAW_P,0.25
+Q_A_RAT_PIT_D,0.007
+Q_A_RAT_PIT_FLTD,11
+Q_A_RAT_PIT_FLTT,11
+Q_A_RAT_PIT_I,0.25
+Q_A_RAT_PIT_P,0.12
+Q_A_RAT_PIT_SMAX,15
+Q_A_RAT_RLL_D,0.007
+Q_A_RAT_RLL_FLTD,11
+Q_A_RAT_RLL_FLTT,11
+Q_A_RAT_RLL_I,0.25
+Q_A_RAT_RLL_P,0.25
+Q_A_RAT_RLL_SMAX,15
+Q_A_RAT_YAW_D,0
+Q_A_RAT_YAW_FLTE,2.5
+Q_A_RAT_YAW_FLTT,11
+Q_A_RAT_YAW_I,0.018
+Q_A_RAT_YAW_P,0.18
+Q_A_RAT_YAW_SMAX,10
+Q_A_RATE_P_MAX,45
+Q_A_RATE_R_MAX,45
+Q_A_RATE_Y_MAX,45
+Q_A_SLEW_YAW,2500
+Q_ANGLE_MAX,1500
+Q_ASSIST_SPEED,18
+Q_FRAME_CLASS,4
+Q_M_BAT_VOLT_MAX,50.4
+Q_M_BAT_VOLT_MIN,39.6
+Q_M_SPIN_ARM,0.07
+Q_M_SPIN_MIN,0.1 # DISABLE_CHECKS: this parameter should be a range restriction, but only values are defined
+Q_M_THST_EXPO,0
+Q_M_THST_HOVER,0.25
+Q_M_YAW_HEADROOM,100
+Q_PLT_Y_RATE,25
+Q_TRANS_DECEL,0.9
+Q_VFWD_GAIN,0.1
+Q_WP_ACCEL_Z,30 # DISABLE_CHECKS: AP limits are based on small quadcopters
+Q_WP_SPEED_DN,200
+Q_WP_SPEED_UP,220
+Q_WVANE_GAIN,1
+RELAY1_FUNCTION,4
+RELAY2_FUNCTION,2
+RELAY2_PIN,108
+RLL_RATE_D,0.001917198
+RLL_RATE_FF,0.7728607
+RLL_RATE_FLTD,11
+RLL_RATE_FLTT,3.183099
+RLL_RATE_I,0.7728607 # DISABLE_CHECKS: it is recommended for RATE_I to match RATE_FF on planes, but the metadata gives a limit of 0.6 for this and 3.0 for FF
+RLL_RATE_IMAX,0.4
+RLL_RATE_P,0.09429314
+RLL_RATE_SMAX,125
+RLL2SRV_RMAX,75
+RNGFND1_GNDCLEAR,27
+RNGFND1_POS_X,0.93
+RNGFND1_TYPE,24
+ROLL_LIMIT_DEG,35  # Set to match NAVL1_LIM_BANK
+RPM1_TYPE,3
+SCALING_SPEED,24
+SERIAL1_BAUD,57
+SERIAL1_PROTOCOL,24
+SERIAL2_BAUD,115
+SERIAL3_BAUD,115
+SERIAL3_PROTOCOL,23
+SERIAL4_BAUD,57
+SERIAL4_PROTOCOL,2
+SERIAL5_PROTOCOL,28
+SERIAL6_BAUD,115
+SERIAL6_PROTOCOL,2
+SERVO_32_ENABLE,1
+SERVO1_FUNCTION,0
+SERVO2_FUNCTION,0
+SERVO3_FUNCTION,0
+SERVO4_FUNCTION,0
+SERVO5_MAX,1950
+SERVO5_MIN,980
+SERVO6_MAX,2000
+SERVO6_MIN,1000
+SERVO7_MAX,2000
+SERVO7_MIN,1000
+SERVO8_MAX,2000
+SERVO8_MIN,1000
+SERVO9_MAX,2000
+SERVO9_MIN,1000
+SERVO17_FUNCTION,33
+SERVO17_MAX,2000
+SERVO17_MIN,1000
+SERVO18_FUNCTION,34
+SERVO18_MAX,2000
+SERVO18_MIN,1000
+SERVO19_FUNCTION,35
+SERVO19_MAX,2000
+SERVO19_MIN,1000
+SERVO20_FUNCTION,36
+SERVO20_MAX,2000
+SERVO20_MIN,1000
+SERVO21_FUNCTION,37
+SERVO21_MAX,2000
+SERVO21_MIN,1000
+SERVO22_FUNCTION,38
+SERVO22_MAX,2000
+SERVO22_MIN,1000
+SERVO23_FUNCTION,39
+SERVO23_MAX,2000
+SERVO23_MIN,1000
+SERVO24_FUNCTION,40
+SERVO24_MAX,2000
+SERVO24_MIN,1000
```

Comment:
Do we need SCR_LD_CHECKSUM and SCR_LD_CHECKSUM ?

Reply:
This PR is very much a placeholder draft. Don't worry about reviewing it in detail yet.